app/controllers/volunteer_ops_controller.rb

```diff
@@ -75,6 +87,27 @@ def build_map_markers(organisations)
     end
   end
 
+  # This needs to be a helper method for the do-it API
+  def collect_all_items (host, href, orgs)
+    n=1
+    while href do
```

Comment:
Style/WhileUntilDo: Do not use `do` with multi-line `while`. (https://github.com/bbatsov/ruby-style-guide#no-multiline-while-do)